src/app/components/common/address_search/addresssearch.css

```diff
@@ -0,0 +1,19 @@
+.error-message {
+    display: none;
```

Comment:
why did you set this property?

if you don't have reason, I recommend you to delete it.